<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>G3PointPlots</class>
 <widget class="QWidget" name="G3PointPlots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G3Point plots</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0" columnstretch="1,0">
   <item row="0" column="0" rowspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="exportCSVToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qG3POINT.qrc">
         <normaloff>:/CC/plugin/G3PointPlugin/images/smallCSVFile.png</normaloff>:/CC/plugin/G3PointPlugin/images/smallCSVFile.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exportImage">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qG3POINT.qrc">
         <normaloff>:/CC/plugin/G3PointPlugin/images/dbImageSymbol.png</normaloff>:/CC/plugin/G3PointPlugin/images/dbImageSymbol.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>560</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qG3POINT.qrc"/>
 </resources>
 <connections/>
</ui>
